openquake/sep/classes.py

```diff
@@ -163,19 +220,289 @@ def compute(self, mag, imt_gmf, sites):
                     mag=mag, pga=gmf, liq_susc_cat=sites.liq_susc_cat,
                     pga_threshold_table=self.pga_threshold_table,
                     return_unit=self.return_unit)
-                vs = hazus_vertical_settlement(
-                    sites.liq_susc_cat, return_unit=self.return_unit)
+                vs = hazus_vertical_settlement(mag=mag,pga=gmf, 
+                    liq_susc_cat = sites.liq_susc_cat, 
+                    settlement_table=self.settlement_table,
+                    return_unit=self.return_unit)
                 out.append(self.deformation_component(ls, vs))
         return out
 
 
-class ZhuLiquefactionGeneral(SecondaryPeril):
+class LateralSpreadNonparametric(SecondaryPeril):
```

Comment:
We should include the names of the authors of the model in the class name, otherwise there's no way of referencing which model the user is specifying in the job file, especially once we start adding more models from other authors.